e2e/testsuite/testsuite.go

```diff
@@ -172,6 +172,28 @@ func (s *E2ETestSuite) SetupChainsRelayerAndChannel(ctx context.Context, channel
 	return r, chainAChannels[len(chainAChannels)-1]
 }
 
+// TODO: Actually setup a single chain.
+// Seeing panic: runtime error: index out of range [0] with length 0 when using a single chain.
+// https://github.com/strangelove-ventures/interchaintest/issues/401
+func (s *E2ETestSuite) SetupSingleChain(ctx context.Context) *cosmos.CosmosChain {
```

Comment:
Unfortunately having to spin up two chains here, even tho the second is essentially discarded. I've opened an issue on interchaintest.